elements/nuxeo-document-actions/nuxeo-document-parent-inspector-button.js

```diff
@@ -0,0 +1,241 @@
+        @apply --layout-justified;
+      }
+    </style>
+
+    <nuxeo-connection id="nxcon" user="{{currentUser}}"></nuxeo-connection>
+
+    <template is="dom-if" if="[[_isAvailable(document)]]">
+      <div class="action" on-tap="_openDialog">
+        <paper-icon-button icon="[[icon]]" noink aria-labelledby="label"></paper-icon-button>
+        <nuxeo-tooltip>[[_label]]</nuxeo-tooltip>
+      </div>
+    </template>
+
+    <nuxeo-dialog id="parent-inspector-dialog" with-backdrop modal>
+      <div class="container">
+        <div class="parentscroll">
+          <div class="scrollable">
+            <div>
+              <h1>Parent Inspector</h1>
+              <div>
+                <p>This is the parent inspector. It displays information about the parent of this document.</p>
+                <table class="table">
+                  <tr>
+                    <td>Tile:</td>
+                    <td>[[document.title]]</td>
+                  </tr>
+                  <tr>
+                    <td>Path:</td>
+                    <td>[[document.path]]</td>
+                  </tr>
+                  <template is="dom-if" if="[[currentUser.isAdministrator]]">
+                    <tr>
+                      <td>UID:</td>
+                      <td>[[document.uid]]</td>
+                    </tr>
+                  </template>
+                </table>
+              </div>
+            </div>
+            <div>
+              <h1>Facets:</h1>
+              <div class="facets">
+                <template is="dom-repeat" items="[[facets]]" as="facets">
+                  <div class="show-items">[[facets]]</div>
+                </template>
+              </div>
+            </div>
+
+            <template is="dom-if" if="[[currentUser.isAdministrator]]">
+              <div>
+                <h1>Schemas:</h1>
+                <div class="schemas">
+                  <template is="dom-repeat" items="[[schemas]]" as="schemas">
+                    <div class="show-items">[[schemas.prefix]]:[[schemas.name]]</div>
+                  </template>
+                </div>
+              </div>
+            </template>
+          </div>
+          <div class="actions">
+            <paper-button
+              noink
+              class="secondary"
+              dialog-dismiss
+              on-tap="_closeDialog"
+              role="button"
+              tabindex="0"
+              animated=""
+              elevation="0"
+              aria-disabled="false"
+              >[[i18n('command.close')]]</paper-button
+            >
+          </div>
+        </div>
+      </div>
+    </nuxeo-dialog>
+  `,
+
+  is: 'nuxeo-document-parent-inspector-button',
+  behaviors: [I18nBehavior, FiltersBehavior],
+
+  properties: {
+    document: {
+      type: Object,
+      notify: true,
+    },
+    icon: {
+      type: String,
+      value: 'icons:info',
+    },
+    layout: {
+      type: String,
+      value: 'parent-inspector',
+    },
+    facets: {
```

Comment:
Since we have no requirement to make this public, we should make it private. We have not concrete wat to do this in vanilla js, but we can either turn this into a read only property or precede the name with an underscore:
```suggestion
    _facets: {
```